src/commands/reminder/reminder-utils.ts

```diff
@@ -0,0 +1,209 @@
+import { Reminder } from '@prisma/client';
+import { getUnixTime, isAfter, isEqual } from 'date-fns';
+import { getPrismaClient } from '../../clients';
+import { OpResult } from '../../utils/opResult';
+
+type SaveReminderInput = {
+  userId: string;
+  guildId: string;
+  message: string;
+  timestamp: number;
+};
+export const saveReminder = async ({
+  userId,
+  guildId,
+  message,
+  timestamp,
+}: SaveReminderInput): OpResult<Reminder> => {
+  try {
+    const currentDate = new Date();
```

Comment:
Need to also convert this to UNIX timestamp before comparision.  Maybe:
`const currentDate = getUnixTime(new Date());`

Reply:
Resolved.